faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -291,6 +291,7 @@ public Builder withCustomHeaders(Map<String, String> headers) {
      * the settings of the {@link Builder} instance.
      */
     public Connection build() {
+      System.setProperty("jdk.httpclient.keepalive.timeout", "3");
```

Comment:
I wonder if there is a better place to execute this statement.
Ideally we only need this once per JVM.
Any ideas?

Reply:
`FaunaClient` can be suitable ?

Reply:
I guess it is more or less the same at the of the day.

We have a use case where we create a lot of clients using `FaunaClient.sessionWith`, so from this perspective the current proposed solution is not too bad.

This is not a blocker for me, just trying to see if we can do only once :+1: